streams/src/main/java/org/apache/kafka/streams/processor/StateStore.java

```diff
@@ -152,4 +153,8 @@ default void init(final StateStoreContext context, final StateStore root) {
         // If a store doesn't implement a query handler, then all queries are unknown.
         return QueryResult.forUnknownQueryType(query, this);
     }
+
```

Comment:
This is a public API change. I will need to update the KIP

Reply:
Thanks! Since it's an added method to an interface people are meant to implement, we need to add a suitable default. I'm planning to take care of that in https://github.com/apache/kafka/pull/11676 (which also adds this method), so no worries.